.github/pull_request_template.md

```diff
@@ -81,12 +81,10 @@ Resolves #00
 
 - [ ] Checked that the design translated visually
 - [ ] Checked behavior. Comment any found issues or broken flows.
-- [ ] Checked keyboard navigability
-- [ ] Checked different states (empty, one, some, error)
-- [ ] Checked for landmarks, page heading structure, and links
 
 #### Validated user-facing changes as a designer
 
+- [ ] Checked keyboard navigability
 - [ ] Checked different states (empty, one, some, error)
```

Comment:
**Question:** Do you want to check for `Checked for landmarks, page heading structure, and links`? Not as familiar for design so not sure if also important

Reply:
Just readded them because I was thinking the same thing - thank you for calling it out! It was an artifact of trying to condense some design checks but I agree that it's important to check these outside of ANDI